src/pages/_app.tsx

```diff
@@ -110,11 +111,12 @@ function AppContent({ Component, pageProps }: AppProps<AppCommonProps>) {
   )
 }
 
-function PostSubscriber() {
+function DatahubSubscriber() {
   const { query } = useRouter()
   const slugParam = (query?.slug || '') as string
   const chatId = getIdFromSlug(slugParam)
-  useSubscribePostsInDatahub(chatId)
+  useDatahubPostSubscriber(chatId)
+  useDatahubModerationSubscriber()
```

Comment:
I guess make sense do some common hook, that will activate all data hub subscribtions in one place, because will have it more and more

Reply:
is it okay if I just move the hook call into subscription-aggregator file?